### PR TITLE
cleanup: cob4-1 with torso and head; cob4-2 without torso and head

### PR DESCRIPTION
**Pull request by floweisshardt**

### CODE REVIEW

File: cob_hardware_config/cob4-2/config/robot_modules.yaml [hunk 1]
```diff
@@ -1 +1 @@
-robot_modules: ['base','torso','head','sensorring']
+robot_modules: ['base']
```
Comment (NadiaHG) [inline cob_hardware_config/cob4-2/config/robot_modules.yaml:1]: why did you delete sensorring? Are not "arm_left", "arm_right",  "gripper_left" and "gripper_right" also necessary?

File: cob_hardware_config/cob4-2/urdf/properties.urdf.xacro [hunk 2]
```diff
@@ -77,19 +77,7 @@
...
-	<!-- ! set these values to make torso straight with (0, 0, 0) joint angles ! -->
-	<property name="torso_1_ref" value="${def_torso_1_ref+offset_torso_1_ref}"/>
-	<property name="torso_2_ref" value="${def_torso_2_ref+offset_torso_2_ref}"/>
-	<property name="torso_3_ref" value="${def_torso_3_ref+offset_torso_3_ref}"/>
-
-	<!-- head reference positions | used as calibration_rising for head axis chain -->
-	<property name="head_1_ref" value="${def_head_1_ref+offset_head_1_ref}"/>
-	<property name="head_2_ref" value="${def_head_2_ref+offset_head_2_ref}"/>
-	<property name="head_3_ref" value="${def_head_3_ref+offset_head_3_ref}"/>
-
```
Comment (NadiaHG) [inline cob_hardware_config/cob4-2/urdf/properties.urdf.xacro:92]: please, update also calibration_offset: https://github.com/ipa320/cob_calibration_data/blob/indigo_dev/cob4-2/calibration/calibration_offset.urdf.xacro

File: cob_hardware_config/cob4-2/config/robot_modules.yaml (hunk 1 above)
Comment (floweisshardt) [inline cob_hardware_config/cob4-2/config/robot_modules.yaml:1]: because this is a configuration file for the old teleop and thus does not support anything else than base

File: cob_hardware_config/cob4-2/urdf/properties.urdf.xacro (hunk 2 above)
Comment (floweisshardt) [inline cob_hardware_config/cob4-2/urdf/properties.urdf.xacro:92]: fixed in https://github.com/ipa320/cob_calibration_data/pull/72

**Comment by NadiaHG** [comment]
Visual review: OK
